app/models/manageiq/providers/embedded_ansible/crud_common.rb

```diff
@@ -55,7 +55,7 @@ def create_in_provider_queue(manager_id, params, auth_user = nil)
     private
 
     def send_notification(op_type, manager_id, params, success)
```

Comment:
Since this is just a save to the local database and less of an external task I wonder if we can just ditch the notifications entirely here?

@Fryguy ?

Reply:
I'm hesitant to remove functionality that was there previously, however I made a similar comment in the previous PR, so I would be fine with ultimately removing it.

Reply:
Resolving this as I think we're going to hold onto it for this pass.

Reply:
@carbonin For the record, we did end up ditching notifications as part of this implementation.

Reply:
Ah, got it. Right, forgot that this handles the repo stuff too.